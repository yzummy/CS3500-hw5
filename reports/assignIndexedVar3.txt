Line 3: Subscript out of bounds					Line 3: Subscript out of bounds
