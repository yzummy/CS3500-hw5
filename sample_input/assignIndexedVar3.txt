{
  A = list(23, 3.14, TRUE, "BoJack");
  A[[5]] = "Mr. Peanutbutter";
  print(A)
}
