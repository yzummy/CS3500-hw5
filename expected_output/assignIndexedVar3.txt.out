Line 3: Subscript out of bounds
